site/guide/README.md

```diff
@@ -24,20 +24,20 @@ The core of the framework is a suite of ultra-simple patterns to describe and or
 -->
 
-At the same time, Frictionless Data is also perfectly capable of powering sophisticated data workflows when used in combination with modern tooling and supporting libraries. This is possible because the framework follows an "atomic" data"[^atomic] -- you can take a minimalistic approach for simple solutions or combine elements for more complex solutions.
+At the same time, Frictionless Data is also perfectly capable of powering sophisticated data workflows when used in combination with modern tooling and supporting libraries. This is possible because the framework follows an "atomic" data[^atomic] -- you can take a minimalistic approach for simple solutions or combine elements for more complex solutions.
 
 If you’d like to learn more about Frictionless before diving in, we created a video walking through the core principles.
 
 <iframe width="730" height="400" src="https://www.youtube.com/embed/lWHKVXxuci0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 <!-- TODO: If you are an experienced data developer [engineer] and want to know how Frictionless compares to other libraries/frameworks, check out the Comparison with Other Frameworks and Tools. -->
 
-[^atomic]: We have borrowed the concept of Atomic Data from the web design field. For us, it means tools or specs are a) broken down into their minimum viable components, and b) these components are combinable into larger and more complex components and systems. The Atomic approach is what underpins the incremental adoptability and the ability to scale from the simplest situation to highly complex data engineering.
+[^atomic]: We have borrowed the concept of Atomic Data from the web design field. For us, it means tools or specs are **a)** broken down into their minimum viable components, and **b)** these components are combinable into larger and more complex components and systems. The Atomic approach is what underpins the incremental adoptability and the ability to scale from the simplest situation to highly complex data engineering.
 
 ### Getting Started
 
 :::warning INFO
-The official guide assumes some basic knowledge about data. If you are new to working with data -- for example, you haven't heard of CSV or JSON or have never used a spreadsheet, it might not be the best idea to jump right into a framework as your first step - grasp the basics, then come back! A good starting point would be the first module, "What is Data?", at [School of Data][scoda].
+The official guide assumes some basic knowledge about data. If you are new to working with data -- for example, you haven't heard of CSV or JSON or have never used a spreadsheet, it might not be the best idea to jump right into a framework as your first step -- grasp the basics, then come back! A good starting point would be the first module, "What is Data?", at [School of Data][scoda].
```

Comment:
This apparently unneeded change is only because this text:
`-- for example, you haven't heard of CSV or JSON or have never used a spreadsheet, it might not be the best idea to jump right into a framework as your first step -`

started with two hypens (which translate to a slightly different Unicode character on the page: `–` vs `-`) but ended only with one.

Reply:
Yes, I saw that later and decided the comment was unnecessary. Thanks!

Reply:
Excellent! That's what I thought since I couldn't find your comment anymore :smile:. But I felt like leaving this explanation in place just in case it's useful for someone else. :slightly_smiling_face: